### PR TITLE
IX: update required site id field to be more flexible

### PR DESCRIPTION
**Pull request by grossjo**

### CODE REVIEW

**Review by guscarreon — CHANGES_REQUESTED** [review]
Can you please add test cases to assert for the correct validation of the json fields? Other adapters include these test in a `params_test.go` file but your adapter is missing its corresponding `adapters/ix/params_test.go`.

**Comment by grossjo** [comment]
@guscarreon tests were added.

File: adapters/ix/params_test.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	`{"siteid":"1234"}`,
+	`{"siteID":"12345"}`,
+	`{"siteId":"123456"}`,
+	`{"siteid":"1234567", "size": [640,480]}`,
+}
+
+var invalidParams = []string{
+	`{"siteid":123}`,
+	`{"siteids":"123"}`,
+	`{"notaparam":"123"}`,
```
Comment (bsardo) [inline adapters/ix/params_test.go:46]: Can you add the following test cases here to verify the min length and oneOf required configuration:
```
`{"siteid":""}`,
`{"siteID":""}`,
`{"siteId":""}`,
`{"siteid":"1234", "siteID":"12345"}`,
`{"siteid":"1234", "siteId":"123456"}`,
```

File: static/bidder-params/ix.json [hunk 2]
```diff
@@ -4,11 +4,21 @@
...
     "siteId": {
       "type": "string",
       "minLength": 1,
       "description": "An ID which identifies the site selling the impression"
     },
+    "siteID": {
+      "type": "string",
+      "minLength": 1,
+      "description": "An ID which identifies the site selling the impression"
+    },
```
Comment (SyntaxNode) [inline static/bidder-params/ix.json:21]: Some hosting companies, such as Xandr, provide a user interface to publishers based on this json schema. To help the UI understand which field to ask the publisher to provide in a case such as this (multiple fields differing only by case) can you please add the word "preferred" to the description of the format you'd like to be the default?

Please see [Criteo's json schema](https://github.com/prebid/prebid-server/blob/master/static/bidder-params/criteo.json) as an example.

Thank you.

File: adapters/ix/params_test.go (hunk 1 above)
Comment (grossjo) [reply adapters/ix/params_test.go:51]: done.

File: static/bidder-params/ix.json (hunk 2 above)
Comment (grossjo) [reply static/bidder-params/ix.json:21]: done.

**Review by bsardo — APPROVED** [review]
LGTM

**Review by guscarreon — APPROVED** [review]
Looks good to me. @grossjo , thank you for addressing our comments

**Pull request merged**